app/src/main/res/xml/quran_preferences.xml

```diff
@@ -48,7 +48,7 @@
         android:key="@string/prefs_sura_translated_name"
         android:persistent="true"
         android:summary="@string/prefs_sura_translated_name_summary"
-        android:title="@string/prefs_sura_translated_name_summary"
+        android:title="@string/prefs_sura_translated_name_title"
```

Comment:
This is a non-related change, but because it is a minor modification, included in this PR.